src/BannerWithLink/index.tsx

```diff
@@ -22,25 +17,22 @@ export type BrazeMessageProps = {
 };
 
 export type Props = {
-    logButtonClickWithBraze: BrazeClickHandler;
-    submitComponentEvent: (componentEvent: OphanComponentEvent) => void;
-    ophanComponentId?: string;
```

Comment:
I think we need to add this back here to reflect that the `ophanComponentId` passed from Braze in the message extras is used by the component.

Reply:
Sorry, I don't understand this comment. The functions have been replaced by the Epic's `trackClick` function - which we agreed was something that had been planned a while back but never implemented? As a consequence the `ophanComponentId` attribute moves into the `brazeMessageProps` object

Reply:
Hey sorry I didn't explain this very well. Having dug a bit more it looks like for this component the `ophanComponentId` would have always come from Braze and so would be in the `brazeMessageProps` just as you said. So removing seems like a good clean up 👍

You probably saw that for the two older banner components before we'd figured out how tracking would work we were falling back on the actual React component name for the `ophanComponentId` (just to give some context on why that's happening).